packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml

```diff
@@ -11,14 +11,14 @@ annotations:
   catalog.cattle.io/requests-memory: 2314Mi
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: istio
-  catalog.cattle.io/upstream-version: 1.13.3
+  catalog.cattle.io/upstream-version: 1.11.8
 apiVersion: v1
-appVersion: 1.13.3
+appVersion: 1.11.8
 description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
   for details.
 icon: https://charts.rancher.io/assets/logos/istio.svg
 keywords:
 - networking
 - infrastructure
 name: rancher-istio
-version: 1.13.3
+version: 100.1.3+up1.11.8
```

Comment:
this version seems incorrect, 100.1.3 should be omitted.

Reply:
Updated the version.